src/api/app/views/webui/shared/_label_form.html.haml

```diff
@@ -2,15 +2,17 @@
   %button.btn.btn-link.dropdown-toggle{ data: { 'bs-toggle': 'dropdown' }, aria: { expanded: 'false' } }
     Edit labels
   .dropdown-menu
-    - existing_labels = labelable.labels.pluck(:label_template_id)
-    = form_for :labels, url: labels_path(project:), method: :put do |f|
-      = f.hidden_field :labelable_type, value: labelable.class
-      = f.hidden_field :labelable_id, value: labelable.id
-      = f.collection_check_boxes(:label_template_ids, project.label_templates, :id, :name) do |l|
-        .dropdown-item
-          .form-check
-            = l.check_box class: 'form-check-input', checked: existing_labels.include?(l.object.id)
-            = l.label class: 'form-check-label' do
-              = render partial: 'webui/shared/label', locals: { label: l.object }
+    - existing_labels = labelable.labels.to_h { |l| [l.label_template_id, l.id] }
```

Comment:
```suggestion
    - existing_labels = labelable.labels.map { |l| [l.label_template_id, l.id] }
```

Reply:
Those two do different things though, we would have to do `map {...}.to_h` if we want to use `map`